locale/en.po

```diff
@@ -1623,40 +1623,64 @@ msgstr "Error"
 msgid "ERROR_FAILED_TO_SAVE_NEW_SETTINGS"
 msgstr "Error: Failed to save new settings to configuration file."
 
-#: ../src/general/PauseMenu.tscn:57
+#: ../src/general/PauseMenu.cs:154
+msgid "RETURN_TO_MENU_WARNING"
+msgstr ""
+"Are you sure you want to exit to the main menu?\n"
+"You will lose any unsaved progress."
+
+#: ../src/general/PauseMenu.cs:170
+msgid "QUIT_GAME_WARNING"
+msgstr ""
+"Are you sure you want to quit to the desktop?\n"
```

Comment:
```suggestion
"Are you sure you want to quit the game?\n"
```
As thrive might in the future be on other platforms, this isn't accurate wording.